anyTV/music-dashboard
# Fix/tunes 148

## A. Summary of Changes

### a. DB Changes:
- N/A

### b. Package Changes:
- none

### c. Implementation Changes:
- Added music provider at the top of sidebar and Back to Tunes at the bottom on use as music provider

## B Reference:

### a. Ticket link/Request Feature link
- [Tunes-148](https://freedom.myjetbrains.com/youtrack/issue/Tunes-148)

### b. Tests Done
- Login as admin/music provider
- Select music provider under Use as Music Provider
- Music provider logo and name should be at the top of the sidebar
- There should be a Back to Tunes link at the bottom of the sidebar
- http://moztrap.freedom.tm/runtests/run/98/env/91/?pagenumber=1&pagesize=20&sortfield=order&sortdirection=asc&filter-suite=44
